.travis.yml

```diff
@@ -15,8 +15,8 @@ install:
 - pip install -r requirements/travis.txt
 script:
 - tox
-after_success:
-- codecov
+#after_success:
+#- codecov
```

Comment:
we might want to create a follow-up PR to remove `.travis.yml` and switch code coverage to GitHub actions too, currently, there's no code coverage report on the PR.